packages/backend/src/daemons/TransactionManager.ts

```diff
@@ -125,8 +125,8 @@ export class TransactionManager {
         const signedData = await this.wallet.signTransaction({
             nonce,
             to,
-            // gasPrice: 2 * 10 ** 9, // 2 gwei
-            gasPrice: 10000,
+            gasPrice: 2 * 10 ** 9, // 2 gwei
+            // gasPrice: 10000,
```

Comment:
The gas price should be 10000 for the optimism goerli testnet. We need to dynamically get this value from the blockchain in the future.